client/settings/general-settings-section/section-heading.js

```diff
@@ -43,16 +42,10 @@ const Title = styled.h4`
 `;
 
 const SectionHeading = () => {
-	const { isUpeEnabled } = useContext( UpeToggleContext );
```

Comment:
Moved to `client/settings/general-settings-section/index.js` (already covered by tests)